### ISSUE TITLE
Numpy issue

### ISSUE THREAD

**Issue by Princeofthebow** [GillesPy2/GillesPy2#241]
Hi,
   I have tried to run a few simple examples and I have run in the following error:

```
Traceback (most recent call last):
  File "C:\Users\Jpk\AppData\Local\Programs\Python\Python37\lib\site-packages\numpy\core\function_base.py", line 117, in linspace
    num = operator.index(num)
TypeError: 'numpy.float64' object cannot be interpreted as an integer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "degradation_example.py", line 121, in <module>
    output = model.run()
  File "C:\Users\Jpk\AppData\Roaming\Python\Python37\site-packages\gillespy2\core\gillespy2.py", line 465, in run
    increment=self.tspan[-1] - self.tspan[-2], **solver_args)
  File "C:\Users\Jpk\AppData\Roaming\Python\Python37\site-packages\gillespy2\solvers\numpy\ssa_solver.py", line 43, in run
    timeline = np.linspace(0, t, (t // increment + 1))
  File "<__array_function__ internals>", line 6, in linspace
  File "C:\Users\Jpk\AppData\Local\Programs\Python\Python37\lib\site-packages\numpy\core\function_base.py", line 121, in linspace
    .format(type(num)))
TypeError: object of type <class 'numpy.float64'> cannot be safely interpreted as an integer.

```

I'm running python 3.7.6 on Windows. It seems to be an issue of Numpy which now considers indexing unsafe in some occastions. 

Can I get over his issue some how?

**Comment by seanebum** [comment]
This issue should be resolved now as of merging #242

**Comment by Princeofthebow** [comment]
Hello, the fix seems to have solver the issue but only when no specific solver is indicated. I ahve selected for instance CythonSSA solver and I get the following error message:
```

Traceback (most recent call last):
  File "/home/jpk/.local/lib/python3.5/site-packages/numpy/core/function_base.py", line 117, in linspace
    num = operator.index(num)
TypeError: 'numpy.float64' object cannot be interpreted as an integer
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/jpk/.local/lib/python3.5/site-packages/IPython/core/interactiveshell.py", line 3326, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-e5949a8c5b14>", line 1, in <module>
    runfile('/home/jpk/Documents/untitled/MylearningGillespy2', wdir='/home/jpk/Documents/untitled')
  File "/home/jpk/Downloads/pycharm-2018.3/helpers/pydev/_pydev_bundle/pydev_umd.py", line 198, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "/home/jpk/Downloads/pycharm-2018.3/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/jpk/Documents/untitled/MylearningGillespy2", line 130, in <module>
    output = model.run(solver=CythonSSASolver)
  File "/home/jpk/.local/lib/python3.5/site-packages/gillespy2/core/gillespy2.py", line 457, in run
    solver_results = solver.run(model=self, t=self.tspan[-1], increment=self.tspan[-1] - self.tspan[-2], **solver_args)
  File "cython_ssa_solver.pyx", line 168, in gillespy2.solvers.cython.cython_ssa_solver.CythonSSASolver.run
  File "<__array_function__ internals>", line 6, in linspace
  File "/home/jpk/.local/lib/python3.5/site-packages/numpy/core/function_base.py", line 121, in linspace
    .format(type(num)))
TypeError: object of type <class 'numpy.float64'> cannot be safely interpreted as an integer.

```

Would there be simpalr problems with other solvers?

**Comment by seanebum** [comment]
I had overlooked the Cython solver when implementing the fix for this. This has been fixed with PR #244 and this error should be resolved for ALL solvers now.  An updated version (1.2.2) has been pushed to PyPI containing these fixes.
